core-tests/src/test/java/org/mule/runtime/core/DefaultMuleMessageSerializationTestCase.java

```diff
@@ -30,7 +32,8 @@
     @Test
     public void testSerializablePayload() throws Exception
     {
-        MuleMessage message = new DefaultMuleMessage(TEST_MESSAGE, muleContext);
+        // TODO MULE-9856 Replace with the builder
```

Comment:
The builder will replace the DefaultMuleMessage instantiations. MutableMuleMessage will remain until all of its uses are replaced with the immutable api.